.github/workflows/release.yml

```diff
@@ -1,53 +1,50 @@
 name: Release
 on:
   workflow_dispatch: {}
-  push:
-    branches:
-      - main
+env:
+  SIGNING_KEY_FILE_PATH: /home/runner/secretKey.gpg
 
 jobs:
   lint:
-    uses: paypal/paypal-messages-android/.github/workflows/lint.yml@release
+    uses: ./.github/workflows/lint.yml
 
   test:
-    uses: paypal/paypal-messages-android/.github/workflows/test.yml@release
+    uses: ./.github/workflows/test.yml
 
   build:
-    uses: paypal/paypal-messages-android/.github/workflows/build.yml@release
-
+    uses: ./.github/workflows/build.yml
+    secrets:
+      SIGNING_KEY_FILE: ${{ secrets.SIGNING_KEY_FILE }}
+      SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
+      SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
+
+  # after build and unit tests are finished, publish all modules at once
+  # to help reduce the probability of failure when interacting with sonatype servers
   release:
-    name: Release
-    needs: [linting, tests, build]
+    needs: [lint, test, build]
+    name: Publish To Sonatype
     runs-on: ubuntu-latest
-
-    permissions:
-      contents: write
-      issues: write
-      pull-requests: write
-
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v3
-        with:
-          # pulls all commits (needed for semantic release to correctly version)
-          fetch-depth: '0'
-          persist-credentials: false
-
-      - name: Set up JDK 17
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set up Java 17
         uses: actions/setup-java@v3
         with:
           java-version: '17'
-          distribution: 'temurin'
-          cache: gradle
-
-      - name: Install Dependencies
-        run: |
-          npm i -g @semantic-release/exec @semantic-release/changelog @semantic-release/git conventional-changelog-conventionalcommits@6.1.0
-
-      - name: GitHub Release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: npx semantic-release@21
-
+          distribution: 'microsoft'
```

Comment:
I'm checking with Steven on the difference but if we're updating the distribution here, we should probably update it everywhere for consistency

Reply:
sounds good